modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts

```diff
@@ -158,7 +157,7 @@ describe("Tests for Outbound Command Event Handler", () => {
     const processSDKOutboundBulkRequestMessageObj = new ProcessSDKOutboundBulkRequestCmdEvt(sampleCommandEventData);
     await producer.sendCommandEvent(processSDKOutboundBulkRequestMessageObj);
 
-    await new Promise(resolve => setTimeout(resolve, 1000));
+    await new Promise(resolve => setTimeout(resolve, 2000));
```

Comment:
If we are going to hard-code a timeout on this, perhaps parameterise the timeout so we can change it in one place?

Reply:
done. added a top file `const`